src/main/java/io/sphere/sdk/meta/SphereClientTuningDocumentation.java

```diff
@@ -3,6 +3,29 @@
 import com.ning.http.client.AsyncHttpClient;
 
 /**
+ <h3 id=blocking-client>Blocking Client</h3>
+ In a lot of frameworks there is no support for asynchronous execution and so it is necessary to wait for the responses.
+ <p>This exmaples lacks a timeout, so in an unfortunate case the thread is blocked forever:</p>
+
+ {@include.example io.sphere.sdk.client.WrongBlockingWithJoin}
+
+ <p>The following examples are even worse because they lack a timeout and they need to deal with checked exceptions:</p>
+
+ {@include.example io.sphere.sdk.client.WrongBlockingWithGetAndSwallowing}
+
+
+ {@include.example io.sphere.sdk.client.WrongBlockingWithGetAndSignature}
+
+ <p>It is better to use the {@link io.sphere.sdk.client.BlockingSphereClient} since it enforces a timeout for resilience
+ and throws directly {@link io.sphere.sdk.models.SphereException}s.</p>
+
+ {@include.example io.sphere.sdk.meta.BlockingClientValueGetDemo}
```

Comment:
I think the current documentation is not stressing enough the separation between the bad/good usage of async.
I would re-order the example:
1st) the correct happy path (using the blocking client)
2nd) a list of bad examples under a title that leaves it clear they are bad examples (I cannot think of a good name right now, sorry).